composer.json

```diff
@@ -8,7 +8,8 @@
         "php": "^7.2",
         "ext-soap": "*",
 
-        "sylius/sylius": "~1.3.0"
+        "sylius/sylius": "~1.3.0",
+        "lsv/pd-parcelshop-api": "dev-master"
```

Comment:
`dev-master` is not an option. It needs to be a stable release.

Reply:
There's no stable, I'll write the owner to make one